app/views/genres/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<% genre = params[:genre] %>
+<h1 class="genre-page-header"><%= genre %> Movies</h1>
+
+<div class="jumbotron">
+  <div class="container">
+    <div class="jumbotron top-3-movies" style="background-color: lightblue">
+      <div class="row">
+      <h3 class="genre-page-header">Top 3</h3>
+        <% genre_top_three(genre).each do |movie| %>
+          <div class="col-sm-6 col-md-4 clickable-card">
+            <%= link_to movie_path(movie.id) do %>
+              <div class="thumbnail">
+                <h3 class="top-movie-title"><%= movie.title %></h3>
+                <%= image_tag("#{movie.movie_poster_url}", alt: "Poster not found...") %>
+                <div class="caption">
+                  <p><%= movie.synopsis %></p>
+                </div>
+              </div>
+            <% end %>
+          </div>
+        <% end %>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
This is (or at least should be) exactly the same as the one on the movie index (homepage), except for how we get the list of movies.  Refactor it out into a partial that takes the movie list as a local so that there's a single source of truth and both will get any updates we make (keeping multiple copies of something in sync is a huge nightmare)!